src/plugins/info/plugin.py

```diff
@@ -91,7 +98,7 @@ def on_config(self, config):
         # hack to detect whether the custom_dir setting was used without parsing
         # mkdocs.yml again - we check at which position the directory provided
         # by the theme resides, and if it's not the first one, abort.
-        if config.theme.dirs.index(get_theme_dir(config.theme.name)):
+        if config.theme.custom_dir:
```

Comment:
Could you explain why this was changed? I hope I didn't miss an explanation in your comments.

Reply:
Since version 1.5.0 MkDocs exposes the `Theme.custom_dir` property: https://www.mkdocs.org/about/release-notes/#upgrades-for-plugin-developers

Before that you had to fetch it with some more or less hacky ways like `Theme._vars` or like you with `Theme.dirs`.

Since I used the `theme.custom_dir` value for validation, I guessed that would be the right way to go to have consistency in fetching the value.

Reply:
Ah, I see! Yes, that makes sense.